style.css

```diff
@@ -0,0 +1,69 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+main{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: 100vh;
+
+    background: #042014;
+
+    animation: animateBg 10s linear infinite;
+}
+@keyframes animateBg{
+    0%{
+        filter: hue-rotate(0deg);
+    }
+    100%{
+        filter: hue-rotate(360deg);
+    }
+}
+main .loader{
+    position: relative;
+
+    width: 120px;
+    height: 120px;
+} 
+main .loader span{
+    position: absolute;
+    inset: 0;
```

Comment:
Is this a shorthand property?

Reply:
yes, top: 0, and left: 0.

Reply:
you can learn more about ***inset*** from here : https://css-tricks.com/almanac/properties/i/inset/